test/es/elasticsearch_helper.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as childProcess from 'child_process';
+import wreck from '@hapi/wreck';
+import { sleep } from '../helper/sleep';
+import {
+  KIBANA_SERVER_USER,
+  KIBANA_SERVER_PASSWORD,
+  ELASTICSEARCH_VERSION,
+  SECURITY_ES_PLUGIN_VERSION,
+} from '../constant';
+
+const PLUGIN_ROOT_DIR = path.resolve(__dirname, '../..');
+const ES_INSTALL_DIR = path.resolve(PLUGIN_ROOT_DIR, '.es');
+let esRootDir: string;
+
+export function downloadElasticsearch(): string {
+  if (fs.existsSync(ES_INSTALL_DIR)) {
+    childProcess.execSync(`rm -rf .es`, { cwd: PLUGIN_ROOT_DIR });
+  }
+  fs.mkdirSync(ES_INSTALL_DIR);
+
+  console.log(`Downloading Elasticsearch...`);
```

Comment:
Please use the single quotes for plain text (non string literal), same comments applies to other occurrence in this page.

Reply:
updated